schema/sh_voices.lua

```diff
@@ -316,15 +314,13 @@ Schema.voices.Add("Combine", "STAR", "Star.", "npc/combine_soldier/vo/star.wav")
 Schema.voices.Add("Combine", "STATIONBLOCK", "Stationblock.", "npc/metropolice/vo/stationblock.wav")
 Schema.voices.Add("Combine", "STAY ALERT", "Stay alert.", "npc/combine_soldier/vo/stayalert.wav")
 Schema.voices.Add("Combine", "STERILIZE", "Sterilize!", "npc/metropolice/vo/sterilize.wav")
-Schema.voices.Add("Combine", "STINGER", "Stinger.", "npc/combine_soldier/vo/stringer.wav")
 Schema.voices.Add("Combine", "STORM", "Storm.", "npc/combine_soldier/vo/storm.wav")
 Schema.voices.Add("Combine", "STRIKE", "Striker.", "npc/combine_soldier/vo/striker.wav")
 Schema.voices.Add("Combine", "SUBJECT 505", "Subject is 505!", "npc/metropolice/vo/subjectis505.wav")
 Schema.voices.Add("Combine", "SUBJECT HIGH SPEED", "All units, be advised, subject is now high-speed!", "npc/metropolice/vo/subjectisnowhighspeed.wav")
 Schema.voices.Add("Combine", "SUBJECT", "Subject!", "npc/metropolice/vo/subject.wav")
 Schema.voices.Add("Combine", "SUNDOWN", "Sundown.", "npc/combine_soldier/vo/sundown.wav")
 Schema.voices.Add("Combine", "SUSPECT INCURSION", "Disp reports suspect-incursion at location.", "npc/metropolice/vo/dispreportssuspectincursion.wav")
-Schema.voices.Add("Combine", "SUSPECT MOVED TO", "Suspect has moved now to...", "npc/metropolice/vo/supsecthasnowmovedto.wav")
```

Comment:
This should be changed to `npc/metropolice/vo/supsecthasmovednowto.wav` (typo included)